<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolunteerCreate</class>
 <widget class="QDialog" name="VolunteerCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>251</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLineEdit" name="leId"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leSurname"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leName"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leBirth"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lePhoneNum"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leEmail"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leAddress"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leActivity"/>
    </item>
    <item>
     <widget class="QLineEdit" name="leStatus"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbCreate">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Create object</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>114</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="lbId">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbSurname">
      <property name="text">
       <string>Surname:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbName">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbBirth">
      <property name="text">
       <string>Data of birth:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbPhoneNum">
      <property name="text">
       <string>Number of phone:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbEmail">
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbAddress">
      <property name="text">
       <string>Address:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbActivity">
      <property name="text">
       <string>Type of activity:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbStatus">
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
